src/main/java/mate/academy/model/Book.java

```diff
@@ -8,8 +8,8 @@
 import java.math.BigDecimal;
 import lombok.Data;
 
-@Entity
 @Data
+@Entity
```

Comment:
```suggestion
@Entity
@Table(name = "books")
```

Don't use `@Data` for entities, find and read articles about it